SDR test failure on main with gdal 3.11

A SDR test has started failing: 
```
AssertionError: these key (actual/expected) errors occured: {'sed_export': (0.08874757588, 0.08896198869), 'usle_tot': (1.86555624008, 1.86480891705), 'avoid_exp': (9179.4052734375, 9203.955078125)}
```
The differences are all by less than 1%, which is reasonable for numerical error in these aggregate results.

It appears to be caused by the update to GDAL 3.11.0.

--- a/constraints_tests.txt
+++ b/constraints_tests.txt
@@ -5,6 +5,3 @@
 # A gdal bug caused our test suite to fail, but this issue is unlikely to
 # occur with regular use of invest. https://github.com/OSGeo/gdal/issues/8497
 GDAL!=3.6.*,!=3.7.*
-
-# https://github.com/natcap/invest/issues/2042
-GDAL<3.11

--- a/constraints_tests.txt
+++ b/constraints_tests.txt
@@ -8,8 +8,3 @@ GDAL!=3.6.*,!=3.7.*
 
 # https://github.com/natcap/invest/issues/2042
 GDAL<3.11
-
-# Pyinstaller 6.10 breaks our windows builds.  Until we can figure out the
-# root cause, let's cap the versions to those that work.
-# https://github.com/natcap/invest/issues/1622
-#pyinstaller<6.10

--- a/constraints_tests.txt
+++ b/constraints_tests.txt
@@ -6,6 +6,9 @@
 # occur with regular use of invest. https://github.com/OSGeo/gdal/issues/8497
 GDAL!=3.6.*,!=3.7.*
 
+# https://github.com/natcap/invest/issues/2042
+GDAL<3.11
+
 # Pyinstaller 6.10 breaks our windows builds.  Until we can figure out the
 # root cause, let's cap the versions to those that work.
 # https://github.com/natcap/invest/issues/1622

--- a/tests/test_sdr.py
+++ b/tests/test_sdr.py
@@ -33,7 +33,7 @@ def assert_expected_results_in_vector(expected_results, vector_path):
         try:
             numpy.testing.assert_allclose(
                 actual_results[key], expected_results[key],
-                rtol=0.00001, atol=0)
+                rtol=0.003, atol=0)
         except AssertionError:
             incorrect_vals[key] = (actual_results[key], expected_results[key])
     if incorrect_vals:

